src/Zabbix/Sender.php

```diff
@@ -32,16 +32,25 @@ class Sender
      */
     protected $response = [];
 
+    /**
+     * Timeout in seconds
+     *
+     * @var float
+     */
+    protected $timeout;
+
     /**
      * Zabbix constructor.
      *
-     * @param string $server Zabbix host
-     * @param int $port Zabbix port
+     * @param string $server  Zabbix host
+     * @param int    $port    Zabbix port
+     * @param float  $timeout Connection timeout 1 second by default
      */
-    public function __construct($server, $port = 10051)
+    public function __construct($server, $port = 10051, $timeout = 1.0)
```

Comment:
Lets use zero timeout for backward compatibility.
You can override default value with 1.0 sec in your application